src/components/ToneHistory/ToneHistory.jsx

```diff
@@ -58,17 +44,14 @@ const ToneHistory = ({ toneResult }) => (
           </LineChart>
         </ResponsiveContainer>
         :
-        <img alt="loading" src="/watson_anim.gif" height="80px" width="80px" />
+          <img alt="loading" src="/watson_anim.gif" height="80px" width="80px" />
       }
     </div>
   </Card>
 );
 
 ToneHistory.propTypes = {
-  toneResult: React.PropTypes.arrayOf(React.PropTypes.shape({
-    text: React.PropTypes.string.isRequired,
-    value: React.PropTypes.number.isRequired,
-  })).isRequired,
+  toneHistory: React.PropTypes.array.isRequired,
```

Comment:
If toneHistory is allowed to be undefined, we would want to remove 'isRequired' otherwise this will likely throw an error when nothing is passed in.